src/plugins/graph/models/graph-layer-model.ts

```diff
@@ -0,0 +1,150 @@
+import { getParentOfType, types } from "@concord-consortium/mobx-state-tree";
+import { typedId } from "../../../utilities/js-utils";
+import { onAnyAction } from "../../../utilities/mst-utils";
+import { DataConfigurationModel, IDataConfigurationModel } from "./data-configuration-model";
+import { getAppConfig } from "../../../models/tiles/tile-environment";
+import { GraphPlace } from "../imports/components/axis-graph-shared";
+import { GraphAttrRole } from "../graph-types";
+import { IUpdateCategoriesOptions } from "../adornments/adornment-models";
+import { GraphModel } from "./graph-model";
+import { IDataSet } from "../../../models/data/data-set";
+import { ISharedCaseMetadata } from "../../../models/shared/shared-case-metadata";
+
+export const GraphLayerModel = types
+  .model('GraphLayerModel')
+  .props({
+    id: types.optional(types.identifier, () => typedId("LAYER")),
```

Comment:
By convention we've been using four-character prefixes, so this would be `LAYR`.